modules/lanelet2/1.2.2.c20250822-f5fc98d/overlay/MODULE.bazel

```diff
@@ -0,0 +1,99 @@
+module(
+    name = "lanelet2",
+    version = "1.2.2.c20250822-f5fc98d",
```

Comment:
NOTE: I put a c in front of the date here so that it will come after the older version `1.2.2.bcr.1`. If I just used the current version + date as other modules do, then this newer version would look to bazel like it is an older version, and 1.2.2.bcr.1 could be chosen over it.